### PR TITLE
Added TLP libraries and buffer to SyncManager.

### PR DESCRIPTION
**Pull request by monday8am**
Task Parallel Library
https://msdn.microsoft.com/es-es/library/hh228603(v=vs.110).aspx

### CODE REVIEW

File: Phoebe/Reactive/SyncManager.cs [hunk 1]
```diff
@@ -96,12 +97,23 @@ public static void Cleanup()
             networkPresence = ServiceContainer.Resolve<INetworkPresence> ();
             client = ServiceContainer.Resolve<ITogglClient> ();
 
+            // TPL block to buffer messages and
```
Comment (alfonsogarciacaro) [inline Phoebe/Reactive/SyncManager.cs]: Is this a "To be continued" comment? ;)

File: Ross/ViewControllers/EditTimeEntryViewController.cs [hunk 2]
```diff
@@ -177,11 +177,18 @@ public override void LoadView()
...
             View = scrollView;
 
             DescriptionEditingMode = false;
         }
 
+        private void backButtonPressed()
+        {
+            Console.WriteLine("here!");
+
+        }
```
Comment (alfonsogarciacaro) [inline Ross/ViewControllers/EditTimeEntryViewController.cs]: Interesting method :)

Comment (monday8am) [inline Ross/ViewControllers/EditTimeEntryViewController.cs]: upps!

File: Phoebe/Reactive/SyncManager.cs [hunk 3]
```diff
@@ -96,12 +97,24 @@ public static void Cleanup()
...
+            }, blockOptions);
+
             StoreManager.Singleton
             .Observe()
 #if __TESTS__
             .SelectAsync(EnqueueOrSend).Subscribe();
 #else
-            .SubscribeQueued(EnqueueOrSend);
+            .Subscribe(processingBlock.AsObserver());
 #endif
```
Comment (alfonsogarciacaro) [inline Phoebe/Reactive/SyncManager.cs]: I run the tests directly with the action block and they're working so we may remove the __TESTS__ compiler directive (L114-116).

**Comment by alfonsogarciacaro** [comment]
I tested it with the simulator (forcing continuation of entries very quickly) and seems to work. We must check it with a physical device.

**Pull request merged**